quarkus.jdt.ext/com.redhat.microprofile.jdt.quarkus.test/src/main/java/com/redhat/microprofile/jdt/quarkus/config/JavaCodeLensQuarkusRestClientTest.java

```diff
@@ -0,0 +1,95 @@
+* Copyright (c) 2021 Red Hat Inc. and others.
+* All rights reserved. This program and the accompanying materials
+* which accompanies this distribution, and is available at
+* http://www.eclipse.org/legal/epl-v20.html
+*
+* Contributors:
+*     Red Hat Inc. - initial API and implementation
+*******************************************************************************/
+package com.redhat.microprofile.jdt.quarkus.config;
+
+import java.io.IOException;
+import java.util.List;
+
+import org.eclipse.core.resources.IFile;
+import org.eclipse.core.runtime.NullProgressMonitor;
+import org.eclipse.core.runtime.Path;
+import org.eclipse.jdt.core.IJavaProject;
+import org.eclipse.jdt.core.JavaModelException;
+import org.eclipse.lsp4j.CodeLens;
+import org.eclipse.lsp4mp.commons.MicroProfileJavaCodeLensParams;
+import org.eclipse.lsp4mp.jdt.core.BasePropertiesManagerTest;
+import org.eclipse.lsp4mp.jdt.core.PropertiesManagerForJava;
+import org.eclipse.lsp4mp.jdt.core.utils.IJDTUtils;
+import org.junit.Assert;
+import org.junit.Test;
+
+import com.redhat.microprofile.jdt.internal.quarkus.providers.QuarkusStaticConfigSourceProvider;
+
+public class JavaCodeLensQuarkusRestClientTest extends BasePropertiesManagerTest {
+
+	@Test
+	public void urlCodeLensYaml() throws Exception {
+		IJavaProject javaProject = loadMavenProject(MicroProfileMavenProjectName.rest_client_quickstart);
+		IJDTUtils utils = JDT_UTILS;
+
+		// Initialize file
+		initConfigFile(javaProject);
+
+		MicroProfileJavaCodeLensParams params = new MicroProfileJavaCodeLensParams();
+		IFile javaFile = javaProject.getProject()
+				.getFile(new Path("src/main/java/org/acme/restclient/CountriesService.java"));
+		params.setUri(javaFile.getLocation().toFile().toURI().toString());
+		params.setUrlCodeLensEnabled(true);
+
+		// No configuration of base url
+		List<? extends CodeLens> lenses = PropertiesManagerForJava.getInstance().codeLens(params, utils,
+				new NullProgressMonitor());
+		Assert.assertEquals(0, lenses.size());
+
+		// /mp-rest/url
+		saveFile(QuarkusStaticConfigSourceProvider.APPLICATION_YAML_FILE, "org:\r\n" + //
+				"  acme:\r\n" + //
+				"    restclient:\r\n" + //
+				"      CountriesService/mp-rest/url: https://restcountries.url/rest", javaProject);
+		assertCodeLenses("https://restcountries.url/rest", params, utils);
+
+		// /mp-rest/uri
+		saveFile(QuarkusStaticConfigSourceProvider.APPLICATION_YAML_FILE, //
+				"org:\r\n" + //
+						"  acme:\r\n" + //
+						"    restclient:\r\n" + //
+						"      CountriesService/mp-rest/url: https://restcountries.url/rest\r\n" + //
+						"      CountriesService/mp-rest/uri: https://restcountries.uri/rest\r\n" + //
+						"", //
+				javaProject);
+		assertCodeLenses("https://restcountries.uri/rest", params, utils);
+	}
+
+	private static void initConfigFile(IJavaProject javaProject) throws JavaModelException, IOException {
+		saveFile(QuarkusStaticConfigSourceProvider.APPLICATION_YAML_FILE, "", javaProject);
```

Comment:
`urlCodeLensYaml` fails, to fix it,add

`saveFile(QuarkusStaticConfigSourceProvider.APPLICATION_PROPERTIES_FILE, "", javaProject);`

Reply:
That's strange CI build is working? But when I execute test in my local labtop, it fails with this previous fix.

Reply:
It works for me without this fix. I'll try quickly on Windows

Reply:
I get test failures on Windows, but not in this file

Reply:
Me i have 2 tests which fails on Windows. This test and ssi test. If you can fix it its nice otherwise merge this pr and i will see that in an another pr

Reply:
Okay, I'll fix the tests on Windows

Reply:
Should be fixed; I added the snippet you suggested.